src/components/OcSelect.vue

```diff
@@ -33,16 +33,6 @@ export default {
       default: null,
     },
   },
-
-  mounted() {
-    this.setComboBoxAriaLabel()
-  },
-  methods: {
-    setComboBoxAriaLabel() {
-      const comboBoxElement = this.$refs.select.$el.querySelector("div:first-child")
-      comboBoxElement.setAttribute("aria-label", this.$gettext("Search for option"))
```

Comment:
instead of removing it it could be a better approach to use it as a fallback label if no label is provided

Reply:
hm i looked up all places where oc-select is used in web and they all set a label (with aria-label) outside of the oc-select, i think it makes more sence to leave the aria-labeling to the implementor since oc-select doesn't include a label.

Reply:
@kulmann @paulcod3 if we plan to merge this it'd be a breaking change and I would suggest cherry-picking it and bringing it inside the `v9.0.0` release this week, too 🍒